SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -466,6 +493,13 @@ void CFlowOutput::SetAnalyzeSurface(const CSolver *solver, const CGeometry *geom
     Tot_Surface_TotalPressure += TotalPressure;
     config->SetSurface_TotalPressure(iMarker_Analyze, TotalPressure);
 
+    for (unsigned short iVar = 0; iVar < nSpecies; iVar++) {
+      su2double Species = Surface_Species_Total[iVar*nMarker_Analyze + iMarker_Analyze];
+      SetHistoryOutputPerSurfaceValue("SURFACE_SPECIES_" + std::to_string(iVar), Species, iMarker_Analyze);
+      Tot_Surface_Species[iVar] += Species;
```

Comment:
Btw: this sum over the iMarkerAnalyze which is also done for eg Temperature is not really correct -> Two Markers with AvgTemp of 300 will lead to the outputted value to be 600 ... So an additional weighting is necessary here